### ISSUE TITLE
Peptide metabolism reactions are reversible

### ISSUE THREAD

**Issue by pranasag** [SysBioChalmers/Human-GEM#838]
#### Current behavior:
Hi all,

The reactions in the subsystem `Peptide metabolism` are all **reversible**, meaning that peptides can be formed from individual amino acids without an ATP cost. Also, I do not think that peptide synthesis even could work this way in animals.
Moreover, a couple of peptide hydrolysis reactions are assigned to other subsystems, this also needs fixing.

#### Curation plan:
1. Set the subsystem of wrongly-annotated reactions (MAR11413, MAR11414, MAR11411, MAR06898, MAR06861, MAR06915, MAR07149, MAR07193, MAR08979) to `Peptide metabolism`;
2. Set all reactions in this subsystems as irreversible.

How does that sound?

**Comment by pranasag** [comment]
Also should note that peptide hydrolysis into individual amino acids itself might have an ATP cost, I would need to dig into literature to be sure.

**Comment by JHL-452b** [comment]
> Set the subsystem of wrongly-annotated reactions (MAR11413, MAR11414, MAR11411, MAR06898, MAR06861, MAR06915,
> MAR07149, MAR07193, MAR08979) to Peptide metabolism;
> Set all reactions in this subsystem as irreversible.

Agree  to change the subsystems and directions of these reactions. But I have not found literature to support that these reactions would cost ATP. For example, the gene ENSG00000124299 ([MAR07193](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR07193); EC:3.4.13.9) is responsible for encoding Xaa-Pro dipeptidase. No ATP was added when the activity of this enzyme was tested in vitro in [this paper](https://febs.onlinelibrary.wiley.com/doi/10.1111/j.1742-4658.2006.05538.x), and [some](https://metacyc.org/reaction?orgid=META&id=3.4.13.9-RXN) [databases](https://www.rhea-db.org/rhea/76407) also show that this reaction does not consume ATP.

**Comment by pranasag** [comment]
That's good because I also checked for evidence on ATP costs and found nothing. So shall I proceed with the fixes?

**Comment by JHL-452b** [comment]
Of curose, please go ahead!

**Comment by JHL-452b** [comment]
I think it is correct to directly modify the above reactions to be irreversible, but it will lead to the appearance of some dead-end metabolites, such as [`Cystyl-Asparaginyl-Methionine`](https://metabolicatlas.org/explore/Human-GEM/gem-browser/metabolite/MAM03527c) (`MAM03527c` in `MAR11028`). These reactions in `Peptide metabolism` were derived from Recon3D and have no gene annotations. I think we can continue to try to find synthetic evidence for these peptides. If there is still no evidence, the rationality of the existence of these metabolites in the model should be considered. What do you think? @pranasag

**Comment by pranasag** [comment]
Well spotted @JHL-452b. In general, I am in doubt regarding di- and tripeptides in the model:
* the description is incomplete I believe (because there should be 20 * 20 different options for dipeptides and 20 * 20 * 20 for tripeptides)
* on the other hand, peptides are present in [blood plasma](https://pubmed.ncbi.nlm.nih.gov/30521147/) and likely are hydrolyzed into free amino acids in the cell, but implementing this thoroughly would be a huge expansion of the model

Not sure how to proceed - but I think that creating dead-ends here is a lesser evil than having artifacts/peptide secretion.

**Comment by edkerk** [comment]
Indeed, I think it is two separate questions:
1. Are the current peptide reactions correct? No -> @pranasag fixed this in #839 
2. Should peptide reactions be in the model at all? If so, should it be extended to cover all possible combinations?
But let's not wait to address question 1 before making a decision on question 2.

**Comment by JHL-452b** [comment]
I found that the HMDB database contains a lot of dipeptide and tripeptide detection information, including the location of the peptides. The following table shows some examples (not all).

| accession   | status     | name                  | Id | direct_parent | location |
|-------------|------------|-----------------------|-----------------|---------------|----------|
| [HMDB0000078](https://hmdb.ca/metabolites/HMDB0000078) | quantified | Cysteinylglycine      | MAM01626        | Dipeptides    |Cytoplasm;Extracellular|
| [HMDB0000721](https://hmdb.ca/metabolites/HMDB0000721) | quantified | Glycylproline         | MAM03630        | Dipeptides    |Cytoplasm (predicted from logP) |
| [HMDB0002335](https://hmdb.ca/metabolites/HMDB0002335) | expected   | Aspartyl-L-proline    | NA              | Dipeptides    |NA |
| [HMDB0003459](https://hmdb.ca/metabolites/HMDB0003459) | quantified | D-Alanyl-D-alanine    | MAM03164        | Dipeptides    | Cytoplasm (predicted from logP) |
| [HMDB0003764](https://hmdb.ca/metabolites/HMDB0003764) | expected   | Glutamylalanine       | NA              | Dipeptides    | NA|
| [HMDB0004207](https://hmdb.ca/metabolites/HMDB0004207) | quantified | Glutamyllysine        | NA              | Dipeptides    | NA|
| [HMDB0004987](https://hmdb.ca/metabolites/HMDB0004987) | detected   | Aspartyllysine        | NA              | Dipeptides    | Cytoplasm (predicted from logP)|
| [HMDB0006248](https://hmdb.ca/metabolites/HMDB0004987) | detected   | gamma-Glutamylalanine | MAM00097        | Dipeptides    | Extracellular|
| [HMDB0006695](https://hmdb.ca/metabolites/HMDB0004987) | quantified | Prolylhydroxyproline  | NA              | Dipeptides    | NA|

If the existing literature provides limited support for the intracellular presence of certain peptides, we can consult the HMDB to determine which peptides should be included in the model and which should be excluded.

**Comment by edkerk** [comment]
Indeed, the rational can then be that "human-GEM contains di- and tripeptides as reported in HMDB", while others not in that database will be removed (unless there is very strong literature evidence of additional peptides?).

In that case, the ones with status "expected" could perhaps be best left out. How many should then be added?

**Comment by JHL-452b** [comment]
> Indeed, the rational can then be that "human-GEM contains di- and tripeptides as reported in HMDB", while others not in that database will be removed (unless there is very strong literature evidence of additional peptides?).

Agree. It would be a great addition.

> In that case, the ones with status "expected" could perhaps be best left out. How many should then be added?

Now I have collected a total of 459 dipeptides or tripeptides in HMDB, of which 274 peptides have the status of `expected`, and the remaining 185 peptides have the status of `detected` or `quantified`. When considering only the 185 `detected` or `quantified` peptides, only 9 peptides are in the model. This means that the remaining 176 peptides need to be added to the model.
